### PR TITLE
Refactor/update to latest coding style

### PR DESCRIPTION
**Pull request by JimDrury**
Refactoring to latest stylistic changes

Changing the digester to be asynchronous using timeout and cancelling of the timeout, so that it only runs when changes have finished being made

Incorporating ESLint

### CODE REVIEW

**Comment by JimDrury** [comment]
The Travis Builds are failling currently due to an NPM INSTALL error with bower


screen shot 2016-07-06 at 09 22 44

File: src/metaFactory.js [hunk 1]
```diff
@@ -1,129 +1,144 @@
...
-        page = newPage;
 
-        pvt.digest();
+        this.reset();
     };
 
 
-    this.reset();
-}
+})(window);
```
Comment (noogn) [inline src/metaFactory.js]: Minor inconsistency - angular is being used in this file but has not been passed in here.

Comment (JimDrury) [inline src/metaFactory.js]: Good shout, totally missed that! Also making same change for document

**Comment by noogn** [comment]
👍

**Pull request merged**